packages/contentstack-migrate-rte/src/lib/util/index.js

```diff
@@ -21,16 +21,18 @@ const { JSDOM } = require('jsdom');
 const collapseWithSpace = require('collapse-whitespace');
 const { htmlToJson } = require('@contentstack/json-rte-serializer');
 const nodePath = require('path');
-const { cliux } = require('@contentstack/cli-utilities');
+const { cliux, managementSDKClient } = require('@contentstack/cli-utilities');
 const packageValue = require('../../../package.json');
 const isBlank = (variable) => {
   return isNil(variable) || isEmpty(variable);
 };
-
-const getStack = (data) => {
+function formatHostname(hostname) {
+  return hostname.split('//').pop();
+}
+async function getStack(data) {
   const tokenDetails = data.token;
-  const client = contentstacksdk.client({
-    host: data.host,
+  const client = await managementSDKClient({
+    host: formatHostname(data.host),
```

Comment:
@vkalta We have removed formatHostName function, take the latest pull from a development branch to get the changes.

Reply:
Done.